src/Accounts.js

```diff
@@ -1,53 +0,0 @@
-import React from 'react';
-import { default as AccountsClient } from 'graphql-accounts';
-import './Wrapper';
-
-const defaultStyle = {
-  wrapper: {
-    minWidth: 500,
-  },
-  title: {
-
-  },
-  fields: {
-
-  },
-  buttons: {
-    wrapper: {
-
-    },
-    button: {
-
-    },
-  },
-};
-
-export { defaultStyle };
-
-const {
-  Wrapper,
-  Title,
-  Fields,
-  Buttons,
-} = AccountsClient.ui;
-
-const Accounts = ({
-  className,
-  style,
-}) =>
```

Comment:
Consider passing React context through to style functions? Since material-ui (and possibly other packages?) use context for theming.